x/xcvm/types/msg.go

```diff
@@ -0,0 +1,87 @@
+package types
+
+import (
+	errorsmod "cosmossdk.io/errors"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+const (
+	TypeMsgSendTransferIntent           = "send_transfer_intent"
+	TypeMsgVerifyTransferIntentProof    = "verify_transfer_intent_proof"
+	TypeMsgTriggerTransferIntentTimeout = "trigger_transfer_intent_timeout"
+)
+
+var _ sdk.Msg = &MsgSendTransferIntent{}
+
+// Type Implements Msg.
+func (MsgSendTransferIntent) Type() string { return TypeMsgSendTransferIntent }
+
+// GetSigners returns the expected signers for a MsgSendTransferIntent message.
+func (msg *MsgSendTransferIntent) GetSigners() []sdk.AccAddress {
+	addr, _ := sdk.AccAddressFromBech32(msg.FromAddress)
```

Comment:
nit: error checking? (or maybe isn't possible here due to interface... I may be forgetting)

Reply:
I looked through their codebase and for all `GetSigners()` implementations they're either swallowing any potential errors from `sdk.AccAddressFromBech32` like I'm doing above, however they also have some cases where they panic if there's an error getting the expected signers. I'll feel like the panic implementation is more appropriate so I'll add it in here instead of swallowing